rollup.config.js

```diff
@@ -23,6 +23,7 @@ const banner = fs.readFileSync(path.join(__dirname, 'licenseBanner.txt'));
 export default {
   input: 'src/dat/index.js',
   output: [{
+    exports: 'named',
```

Comment:
See https://rollupjs.org/guide/en#output-exports-exports for docs on this option. Without it, rollup warns we're mixing named and default exports, and this option basically says "yeah that's fine".